### PR TITLE
Better documentation on the use of kpt pkg sync set

### PR DESCRIPTION
**Pull request by mortent**
The documentation of `sync` doesn't contain any links to the description of the `sync set` command. This adds this link and adds some clarification on how to use the `sync set` command.

Ref #308

@pwittrock

### CODE REVIEW

File: internal/docs/generated/livedocs/docs.go [hunk 1]
```diff
@@ -91,7 +91,6 @@ functionality for polling a set of resources and computing the aggregate status
 for the set. For CRDs, there is a set of recommendations that if followed, will allow
 kpt live apply to correctly compute status.
 
-###
```
Comment (pwittrock) [inline internal/docs/generated/livedocs/docs.go]: Keep this, it is needed by the parser to not show these links in the built-in docs.

Comment (mortent) [reply internal/docs/generated/livedocs/docs.go]: I have updated the PR.

**Comment by pwittrock** [comment]
plz resolve conflicts

**Comment by mortent** [comment]
Resolved conflict

**Pull request merged**